ow_faults/src/FaultInjector.cpp

```diff
@@ -22,6 +22,12 @@ FaultInjector::FaultInjector(ros::NodeHandle node_handle)
   m_fault_power_state_of_charge_pub = node_handle.advertise<std_msgs::Float64>("temporary/power_fault/state_of_charge", 10);
   m_fault_power_temp_pub = node_handle.advertise<std_msgs::Float64>("temporary/power_fault/temp_increase", 10);
 
+  //antenna fault publishers and subs
+  m_fault_ant_pan_sub = node_handle.subscribe("/_original/ant_pan_position_controller/command", 3, &FaultInjector::antennaePanFaultCb, this);
+  m_fault_ant_tilt_sub = node_handle.subscribe("/_original/ant_tilt_position_controller/command", 3, &FaultInjector::antennaeTiltFaultCb, this);
```

Comment:
Please shorten these two lines -- they are really too long, and will get longer if you refactor the callback functions.  Shortening the long (>80-90 column) lines before and after would be nice, but don't worry if it's a hassle.  I realize they contain long names.

Reply:
I dont think the names could be shortened without losing info unless we assume people would know pan and tilt only applies to the antenna?

Reply:
Agreed, though I was just requesting shorting the _lines_ if feasible, e.g. line breaks between tokens or separators.

Reply:
I see, yea in that case I can break apart the lines to make it easier to read then